runtime/serde/serde-cbor/common/test/aws/smithy/kotlin/runtime/serde/cbor/CborDeserializerSuccessTest.kt

```diff
@@ -559,7 +549,7 @@ class CborDeserializerSuccessTest {
         val actual = deserializer.deserializeList(SdkFieldDescriptor(SerialKind.List)) {
             val list = mutableListOf<UByte>()
             while (hasNextElement()) {
-                list.add(deserializeByte().toUByte())
+                list.add(deserializeLong().toUByte())
             }
             return@deserializeList list
```

Comment:
**Question**: Why did so many tests from deserializing narrow types (e.g., `Byte`) to deserializing `Long`?

Reply:
Tests of this form started failing after adding the out-of-bounds check for number deserialization because the numeric value is 255 which doesn't fit in a Byte. Trying to call `deserializeByte` here would result in the out-of-bounds check throwing an exception, so it's updated instead to `deserializeLong`